LPS-170861 CI Test

JoyceWang08211 opened the pull request:
https://issues.liferay.com/browse/LPS-170861

Commit:
LPS-157406 Integration tests.

For LPS-152122.

Commit:
LPS-165445 Replace IndexerRequestBufferExecutorWatcher with IndexerRequestBufferExecutor (because the class IndexerRequestBufferExecutor is an internal interface, and there is only one implementation, it can be directly referenced.)

BryanEngler committed:
LPS-171359 portal-search-elasticsearch7-impl: Default to original behavior of 0 retires. Improve log messages

https://issues.liferay.com/browse/LPS-171359

Commit:
LPS-152126 Use metatype defaults when companyId=0 configuration is deleted.

For LPS-152122.

Commit:
LPS-157406 Align asserted errors with upstream wordsmiting.

For LPS-152122.

Commit:
LPS-165445 Remove unused configuration

Commit:
LPS-165445 Apply

Commit:
LPS-152126 Follow up to 5b271748bb. Better if we use individual ExceptionMappers to avoid registering one for PortalException?

For LPS-152122.

Commit:
LPS-150933 Remove release ignore for LPS-152122.

For LPS-152122.

dantewang committed:
LPS-165445 SF, return value not used

Commit:
LPS-165445 SF, inline(IndexerRequestBufferExecutor cannot directly reference IndexWriterHelper, there is a circular reference problem {com.liferay.portal.search.internal.IndexWriterHelperImpl[_searchPermissionChecker] -> com.liferay.portal.search.internal.SearchPermissionCheckerImpl[_indexerRegistry] -> com.liferay.portal.search.internal.IndexerRegistryImpl[setIndexerRequestBufferOverflowHandler] -> com.liferay.portal.search.internal.buffer.DefaultIndexerRequestBufferOverflowHandler[_indexerRequestBufferExecutor] -> com.liferay.portal.search.internal.buffer.IndexerRequestBufferExecutor[indexWriterHelper] -> (circular reference)}, see f338e9e8d4ed6a08aaab001b67e0a896149d04da)

Commit:
LPS-165169 Remove unused method stream()

Commit:
LPS-169785 Clean up unnecessary spring-extender dependencies

Commit:
LPS-167050 dispatch-service - validate type after name validation (slight change required to make code clear)

Commit:
LPS-167050 dispatch-test - make sure tests use valid dispatch executor types

Commit:
LPS-167050 dispatch-service - service.xml

Commit:
LPS-167050 dispatch-service - throw exception generated by service builder

Commit:
LPS-167050 dispatch-test - test DispatchTriggerTypeException

qz1010 committed:
LPS-169488 Rename attribute

jeffwu0724 committed:
LPS-169043 add commons-collections4 to source-formatter with version 4.4. We are using 4.2, 4.3, 4.4 in the system now, chose version 4.4, which is the highest version for better maintainance

Commit:
LPS-170042 dispatch-service - change single-node check expression to cover MEMORY-CLUSTERED and MEMORY-PERSISTED single node modes, update service to force single node memory persisted if requested mode differs from annotation given mode

Commit:
LPS-170114 Remove immediate from Portal Scheduler module

Valmir-unicap committed:
LPS-166350 New Corrected the width to 25% and added the truncate an tooltip in text and select area.

Commit:
LPS-167050 salesforce-connector - make sure talend executor is activated before deploy starts (this also fixes flakiness in CI environment)

Commit:
LPS-165445 Merge IndexerRequestBufferExecutor and BaseIndexerRequestBufferExecutor into DefaultIndexerRequestBufferExecutor and remove property setting as it is the only implementation

Commit:
LPS-165445 Merge DefaultIndexerRequestBufferOverflowHandler and IndexerRequestBufferOverflowHandler

Commit:
LPS-165445 Apply

Commit:
LPS-167050 validate dispatchTaskExecutorType

Commit:
LPS-169456 ObjectDefintion The PUT method should only require the same fields as the POST method

jeffwu0724 committed:
LPS-170748 move org.jfree to shared-dependencies

jeffwu0724 committed:
LPS-170748 manullay change compileInclude to compileOnly for org.jfree, since we have moved it to shared-dependencies

Commit:
LPS-165169 Simplify, remove redundant method references in class SearchRequestContributorsRegistryImpl

Commit:
LPS-165169 Inline and remove unused classes

JoyceWang08211 committed:
LPS-162107 Update macro

JoyceWang08211 committed:
LPS-162107 Add macro to view saml configurations

JoyceWang08211 committed:
LPS-162107 Add test to import JSON file to configure SAML SP

JoyceWang08211 committed:
LPS-162107 Add test to import JSON file to configure SAML IDP

JoyceWang08211 committed:
LPS-162107 Add dependencies

adolfopa committed:
LPS-170626 Use dropdown items instead of menu item

adolfopa committed:
LPS-170626 Add icons to dropdown items

jeffwu0724 committed:
LPS-169043 export commons-collections4 from shared-dependencies-apache-commons with version 4.4

deborasoliveira committed:
LPS-170569 Add KeywordCanDisplayInSemibold Testcase

Commit:
LPS-165445 buildLang

JoyceWang08211 committed:
LPS-162107 Add upgrade tests

jeffwu0724 committed:
LPS-169043 auto sf

jeffwu0724 committed:
LPS-169043 manually change compileInclude to compileOnly for commons-collections4, because we can refer it from shared-dependencies-commons

jeffwu0724 committed:
LPS-169043 remove commons-collections4 from -includeresource in batch-engine-test, dynamic-data-lists-test and content-dashboard-test, because we can get the reference from shared-dependencies

JoyceWang08211 committed:
LPS-162107 Move to proper folder

JanisZhang committed:
LPS-170769 Remove optional<> usages in FragmentEntryProcessorContext and FragmentRendererContext

JanisZhang committed:
LPS-170769 Apply usages

JanisZhang committed:
LPS-170769 Remove optional<> usages in MenuDisplayFragmentConfiguration

JanisZhang committed:
LPS-170769 Apply usages

Commit:
LPS-170771 Convert method reference to field reference(Because property ReferencePolicyOption.GREEDY exists, I did not directly reference AMImageEntryProcessor)

liuxingye committed:
LPS-166586 Add CanFilterEntriesUsingRelationshipField test

liuxingye committed:
LPS-166589 Add CanSeeEntriesWithFilterApplied test

Commit:
LPS-170114 Remove service property and let it be implied as immediate

ambrinchaudhary committed:
LPS-170626 Remove nesting level. This is only necessary when there are groups in dropdowns, and here is not the case

aline-doleron committed:
LPS-166350 Do not show empty tooltip

javalosjr96 committed:
LPS-170857 Bump google.cloud:libraries-bom to 26.1.5

BND was also modified for new imports.

JoyceWang08211 committed:
LPS-162107 TF

JanisZhang committed:
LPS-170769 Remove Stream usages in BaseEditableValuesConfigurationExportImportContentProcessor

aline-doleron committed:
LPS-166350 Update snapshots

Commit:
LPS-157406 Auto SF (dependency micro update)

For LPS-152122.

deborasoliveira committed:
LPS-170554 Add CanDisplayAllAttributes Testcase

deborasoliveira committed:
LPS-170554 Adjusting teardown for the new tests

ealonso committed:
LPS-170962 baseline

deborasoliveira committed:
LPS-170554 SF

ces-totrinh committed:
LPS-171030 Elasticsearch7 adds field mapping for userGroupRoleIds

ealonso committed:
LPS-170962 No needed return

austinchiang committed:
LPS-170179 Split into two tests

ealonso committed:
LPS-170962 Auto

ealonso committed:
LPS-170962 Adds supports to filter by ddm structure key

ealonso committed:
LPS-170962 Update method calls

ealonso committed:
LPS-170962 Expose methods to filter by ddmStructureKey

ealonso committed:
LPS-170962 Uses new method

austinchiang committed:
LPS-170179 Flag as e2e tests

austinchiang committed:
LPS-170179 Remove unnecessary test

austinchiang committed:
LPS-170179 Update test names

austinchiang committed:
LPS-170179 Remove unnecessary step

ealonso committed:
LPS-170962 It should be JournalArticleItemSelectorReturnType

austinchiang committed:
LPS-170179 Update test

austinchiang committed:
LPS-170179 Use more efficient navigation

austinchiang committed:
LPS-170179 Rename test

austinchiang committed:
LPS-170179 Remove test covered by existing tests

austinchiang committed:
LPS-170179 Sort

sandrodw3 committed:
LPS-170968 Make close button optional

sandrodw3 committed:
LPS-170968 Add header when active item is editable

ces-binhtran committed:
LPS-168587 The configuration method should be the same name with the input on rss-settings taglib

sandrodw3 committed:
LPS-170968 Restore previous panel when it's valid

sandrodw3 committed:
LPS-170968 Add comments so panel handling is more clear

sandrodw3 committed:
LPS-170968 buildLang

sandrodw3 committed:
LPS-170968 Apply secondary color to fragment comments back button

jackelyneaquino committed:
LPS-168479 Adding roles prior to permissions in order of invoke

LPS-168479 Assert Role Permission

LPS-168479 Assert role permission corrected

jeffwu0724 committed:
LPS-170748 auto generate, by running "ant build-lib-versions"

jeffwu0724 committed:
LPS-169043 auto generate, by running "ant build-lib-versions"

kevenleone committed:
LPS-171101 Create Dispatch Trigger Impl

JanisZhang committed:
LPS-170769 Remove Stream usages in BaseFragmentCollectionContributor

Commit:
LPS-171024 Convert method reference to ServiceTrackerMap(Because 'BulkReindexer' is not a '@ProviderType' interface, I do not directly reference the implementation class 'UserBulkReindexer')

Commit:
LPS-171024 Fix the test

ricardocousolr committed:
LPS-168169 Lower numerical value of priority means higher precendence

Entries with no priority (null) before are set to -1 in step v1_7_0, so
they have the most precendence. To respect this we must order by
priority asc too. Entries with the same priority value are sorted by
createDate (more recent means more precedence).

sandrodw3 committed:
LPS-171029 Render header even if there's no any panel

sandrodw3 committed:
LPS-171029 Add tests ItemConfiguration component

Commit:
LPS-149027 Creating the file and adding stubs for DXP Cloud activation form already submitted scenario

Commit:
LPS-149027 Add CanViewAUsefulMessage test

Commit:
LPS-149027 Auto SF

achaparro committed:
LPS-168721 Always show a message in the Upgrade Report

kevenleone committed:
LPS-171101 Update Object Definition

kevenleone committed:
LPS-171113 Call Dispatch Trigger after Create the Task

kevenleone committed:
LPS-171112 Update Status

kevenleone committed:
LPS-171112 SF

kevenleone committed:
LPS-171112 Enforce Type

evanilson-santana committed:
LPS-166585 Add CanCreateFilterUsingRelationshipField Test

evanilson-santana committed:
LPS-166587 Add CanEditFilterWithRelationshipField Test

deborasoliveira committed:
LPS-170569 Add ClayAlertToast path file

deborasoliveira committed:
LPS-170569 Adjusting the tasks of KeywordCanDisplayInSemibold Testcase

deborasoliveira committed:
LPS-170554 Adjusting the tasks of CanDisplayAllAttributes Testcase

deborasoliveira committed:
LPS-170554 SF

deborasoliveira committed:
LPS-170554 Add ClayAlert.path file for the static alert paths

ealonso committed:
LPS-170962 Adapt logic to match with other asset selectors, like EditAssetListEntryManualDefaultPropsTransformer.js

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

JoyceWang08211 commented:
ci:test:saml

liferay-continuous-integration commented:
<html><h3>:x: ci:test:saml - 0 out of 2 jobs passed in 55 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4117">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1377401_7080</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/0abd25cbed9fcb90f3dbe008036fa8a8e068d04f">0abd25cbed9fcb90f3dbe008036fa8a8e068d04f</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c33512e8dcdd2be3baefcfc741dd9e55b484dc53">c33512e8dcdd2be3baefcfc741dd9e55b484dc53</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2129401528">EE Development Acceptance (master) - 4115 - 2022-12-14[12:41:28]</a></p><details><summary><strong>ci:test:saml - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7080/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/341827/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7080/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/341827//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/341827//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort_">LocalFile.SAML#ImportJSONFileToConfigureSAMLSPOnNonDefaultPort</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7080/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7080/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7080/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7080/">test-portal-acceptance-pullrequest(master)#7080</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7080/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>saml<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/141">JoyceWang08211#141</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2129838434">[master] ci:test:saml - JoyceWang08211 &gt; JoyceWang08211 - PR#141 - 2022-12-14[19:55:05]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/16368/">publish-testray-report#16368</a><br/></div></div>

JoyceWang08211 commented:
ci:test:saml

liferay-continuous-integration commented:
<html><h3>:x: ci:test:saml - 0 out of 2 jobs passed in 52 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4117">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1327401_7416</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/0abd25cbed9fcb90f3dbe008036fa8a8e068d04f">0abd25cbed9fcb90f3dbe008036fa8a8e068d04f</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c33512e8dcdd2be3baefcfc741dd9e55b484dc53">c33512e8dcdd2be3baefcfc741dd9e55b484dc53</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2129401528">EE Development Acceptance (master) - 4115 - 2022-12-14[12:41:28]</a></p><details><summary><strong>ci:test:saml - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/7416/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/147650/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7416/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/147650//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/147650//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort_">LocalFile.SAML#ImportJSONFileToConfigureSAMLSPOnNonDefaultPort</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-32/test-portal-acceptance-pullrequest(master)/7416/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-32/test-portal-acceptance-pullrequest(master)/7416/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-32/test-portal-acceptance-pullrequest(master)/7416/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/7416/">test-portal-acceptance-pullrequest(master)#7416</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7416/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>saml<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/141">JoyceWang08211#141</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2130191500">[master] ci:test:saml - JoyceWang08211 &gt; JoyceWang08211 - PR#141 - 2022-12-14[22:43:24]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-32.liferay.com/job/publish-testray-report/18571/">publish-testray-report#18571</a><br/></div></div>

JoyceWang08211 commented:
ci:test:saml

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>saml</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/6236/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6236/jenkins-report.html">here</a>.</b></p>

sirfabios committed:
LRQA-78596 Fix ProcessBuilderKaleoDesignerReact#CanAddScriptToDefineAction

sirfabios committed:
LPS-169465 Test fix ProcessBuilderKaleoDesignerReact#CanDeleteAction

andrea-ale-sbarra committed:
COMMERCE-10471 changed cookie type

evanilson-santana committed:
LPS-168858 Update groovy script with new object id name

evanilson-santana committed:
LPS-168858 add an expression on the edit task to validate the action was edited

sirfabios committed:
LPS-170067 Add WorkflowMetrics tests in quarantine

manuferreira committed:
LRAC-12356 Add xpath to track the event analysis dropdown

manuferreira committed:
LRAC-12356 Add macro to select the autocomplete option in the event analysis

manuferreira committed:
LRAC-12356 Update addFilter macro to satisfy another condition

manuferreira committed:
LRAC-12356 Add CheckResultNotAppearIfAnyAttributeConditionsNotContainedValue testcase

manuferreira committed:
LRAC-12356 Update test to use this macro only once

manuferreira committed:
LRAC-12356 Add PaginateDataSourceList testcase

manuferreira committed:
LRAC-12356 Remove unused line

manuferreira committed:
LRAC-12356 SF

patriciajeanperez committed:
LPS-164328 Remove search step in User macro

julschong committed:
LPS-171276 Remove Stream usages and use for loop instead

julschong committed:
LPS-171276 Auto, SF

tinatian committed:
LPS-165169 SF, simplify

dantewang committed:
LPS-170769 SF

shuyangzhou committed:
LPS-171360 Remove NotificationEventFactory, just create NotificationEvent directly.

shuyangzhou committed:
LPS-171360 Remove notifications channel infrastructure which is no longer used anywhere.

shuyangzhou committed:
LPS-171360 Semantic versioning

shuyangzhou committed:
LPS-171357 Avoid using StoreFactory from component.

shuyangzhou committed:
LPS-171357 Extract ConvertProcess as components

shuyangzhou committed:
LPS-171357 Explicitly checking store readiness

shuyangzhou committed:
LPS-171357 Merge StoreFactory into DLStoreImpl

shuyangzhou committed:
LPS-171357 Semantic versioning

markocikos committed:
LPS-167272 Update alloy-ui to 3.1.0-deprecated.108

Commit:
LPS-167050 dispatch - Autogenerated

Commit:
LPS-167050 dispatch - Autogenerated - baseline

Commit:
LPS-171323 Pagination fix

Commit:
LPS-171324 Add Styles To Dashboard Charts

evanilson-santana committed:
LPS-169427 Update ERC field validation from integer to string on the expression builder

riccardo-ferrari committed:
COMMERCE-10497 commerce-product-service Make CommerceChannelIndexer permission filtering configurable

Commit:
LPS-169382 Add DeleteWCFolderWithinPublication testcase

shuyangzhou committed:
LPS-171357 Fix test

CalumR23 committed:
POSHI-459 Generate CHANGELOG

CalumR23 committed:
POSHI-459 Update default poshi runner version

CalumR23 committed:
POSHI-459 Use latest gradle-plugins-poshi-runner

CalumR23 committed:
POSHI-459 Update poshi-core version

Commit:
LPS-170835 Add filter functionality by clicking on chart

interaminense committed:
LRAC-12601 Fix commerce channels total when toggle commerce sync

itstali committed:
LPS-171339 Fix RaylifeHomePage#ZipCodeIsRequired Test

ealonso committed:
LPS-170962 Order list by label

Commit:
LRAC-12627 Modify the branch that is used in AC acceptance tests

danielCamara02 committed:
LPS-171134 Add CanApprovePageInAPublication test

interaminense committed:
LRAC-12601 Do no use uid because ids are generated every time table is rendered

interaminense committed:
LRAC-12601 Update generated types

kiwm committed:
LPS-171345 Change the display logic on Attachment.tsx function File

rbohl committed:
LPS-166942 wordsithing

kevenleone committed:
LPS-170826 Add Revalidate function

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

kevenleone committed:
LPS-170826 Add Rest Implementations for Task, Subtask, CaseResult and TaskUsers

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

kevenleone committed:
LPS-170826 Simplify

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

kevenleone committed:
LPS-170826 Update Outlet parameters and standardize

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

kevenleone committed:
LPS-170826 Use Outlet  with new properties on Testflow and Subtask

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

kevenleone committed:
LPS-170826 Update Outlet Router

Co-authored-by: Rafael Ferreira <rafael.ferreira@liferay.com>

tinatian committed:
LPS-165445 Baseline

pei-jung committed:
LPS-171336 Implements Clusterable so that it will run on every node in cluster

ealonso committed:
LPS-171352 As message said, we need to keep this line here

david-truong committed:
LPS-164162 fix css styles not applied on content pages in a publication

ealonso committed:
LPS-169585 Removes no necessary custom css

ealonso committed:
LPS-169585 It should be namespace

ealonso committed:
LPS-169585 Uses taglib

ealonso committed:
LPS-169569 Removes no needed custom css and classes

ealonso committed:
LPS-169569 SF

ling-alan-huang committed:
LPS-168928 Auto SF

ealonso committed:
LPS-169576 Removes unnecessary css and classes

ealonso committed:
LPS-169576 SF

ealonso committed:
LPS-169576 SF - simplify

ealonso committed:
LPS-169598 Removes unnecessary css and classes

ealonso committed:
LPS-169596 Removes unnecessary css

ealonso committed:
LPS-169529 Update classes and removes unnecessary css file

Commit:
LRQA-78859 Fix typo

JanisZhang committed:
LPS-170769 semver

Commit:
LPS-171366 Fix typo

brianchandotcom committed:
LPS-171276 SF

brianchandotcom committed:
artifact:ignore feature-flag-web 1.0.2 prep next

JoyceWang08211 commented:
ci:stop

JoyceWang08211 commented:
ci:test:saml

liferay-continuous-integration commented:
<html><h3>:x: ci:test:saml - 0 out of 2 jobs passed in 44 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4127">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1377401_7200</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/55c4faf9f518c2d7a48dbcd9bbfe7ededf3697b1">55c4faf9f518c2d7a48dbcd9bbfe7ededf3697b1</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/4d1457b122561ece9ed5e6e1528c8b88c1bdb701">4d1457b122561ece9ed5e6e1528c8b88c1bdb701</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2132963837">EE Development Acceptance (master) - 4126 - 2022-12-17[05:09:05]</a></p><details><summary><strong>ci:test:saml - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7200/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/151451/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7200/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/151451//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/151451//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort_">LocalFile.SAML#ImportJSONFileToConfigureSAMLSPOnNonDefaultPort</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7200/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7200/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7200/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7200/">test-portal-acceptance-pullrequest(master)#7200</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7200/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>saml<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/141">JoyceWang08211#141</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2133971522">[master] ci:test:saml - JoyceWang08211 &gt; JoyceWang08211 - PR#141 - 2022-12-18[23:50:56]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/16651/">publish-testray-report#16651</a><br/></div></div>

sandrodw3 committed:
LPS-170872 Sort

victorg1991 committed:
LPS-171099 Add label to the portlet options tag

4lejandrito committed:
LPS-169781 Using local services

As in f6187cd26a98099ba86c801436cfa7eb0af961df that fixed this same
problem in the old dto converter.

victorg1991 committed:
LPS-171107 Remove unused role

brianchandotcom committed:
LPS-170769 SF

brianchandotcom committed:
artifact:ignore frontend-taglib-util-freemarker-contributor 5.0.12 prep next

pedro-oliveira446 committed:
LPS-170932 add new function to extract the requestId from url

Commit:
LPS-169892 Add CanOrderCategoryFacetTerms testcase

ealonso committed:
LPS-170962 Rename

ealonso committed:
LPS-170962 Auto

brianchandotcom committed:
POSHI-459 prep next

brianchandotcom committed:
POSHI-459 prep next

brianchandotcom committed:
POSHI-459 prep next

brianchandotcom committed:
Baseline

brianchandotcom committed:
LPS-168858 #sfme

brookedalton committed:
LPS-171134 Move to publications workflow testcase file

Commit:
LPS-171227 Add Feature Flag to LPS-165850

patriciajeanperez committed:
Revert "LPS-170474 SF"

This reverts commit 27115da6a995b20d3b032aa2379d0ba4379eef42.

patriciajeanperez committed:
Revert "LPS-170474 Restore PrincipalThreadLocal to original state"

This reverts commit f6e6919430406792fc94e5a1fb0264726c241b7d.

Commit:
LPS-171227 afcb

brianchandotcom committed:
LPS-167050 SF

brianchandotcom committed:
LPS-170967 Wordsmith

brianchandotcom committed:
LPS-170967 Regen

itstali committed:
LPS-171349 Fix path

manuferreira committed:
LRAC-12356 Update test to see if the list of data sources are not on the second page

michaelhashimoto committed:
LRCI-3362 Run 7.4 Hotfixes against the base portal git id

Luizcarlosqueiroz committed:
LPS-169212 Prevent failure by waiting error message disappear

Luizcarlosqueiroz committed:
LPS-169212 Add back function to allow data to be saved

marcelabc committed:
LPS-171337 We must not populate the values map if the ddmFormField is null

calvinctvo committed:
LRAC-12526 Remove tests from quarantine

Luizcarlosqueiroz committed:
LPS-169212 Add view alert

rafaprax committed:
LPS-171442 Make Opcodes.V1_6 the default option instead of the Opcodes.V1_2 and patch mvel2

rafaprax committed:
LPS-171442 Update usages

brookedalton committed:
LPS-169382 Rename test

rafaprax committed:
LPS-171442 ant build-lib-versions

john-co committed:
LPS-170569 Refactor test

john-co committed:
LPS-170569 Remove duplicate path

xbrianlee committed:
LPS-171445 Close iframe instead of navigating back to page

michaelhashimoto committed:
LRCI-3362 Ensure the correct portal repository name is used when using release tags

WYZ-Issac committed:
LPS-170626 Update the locator to find the ellipsis button

Jeremy-Ch committed:
LPS-171365 Update macro to check the CSS style

Jeremy-Ch committed:
LPS-171365 Update tests to use the correct variable name

Jeremy-Ch committed:
LPS-171365 Remove paths that are no longer needed

ling-alan-huang committed:
POSHI-450 Auto SF

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPS-171442 prep next

brianchandotcom committed:
LRCI-3362 prep next

xbrianlee committed:
Revert "LPS-166350 Update snapshots"

This reverts commit 8d8480c1d3e8ad477ec952de25ca3a7dca5e343e.

xbrianlee committed:
Revert "LPS-166350 Do not show empty tooltip"

This reverts commit 4892ee2ae322c994923e0d668e2daa811438d8b2.

xbrianlee committed:
Revert "LPS-166350 New Corrected the width to 25% and added the truncate an tooltip in text and select area."

This reverts commit 430ea18f82dae45aef95e2408b2600481f333b7f.

JoyceWang08211 commented:
ci:test:bundle

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:bundle - 1 out of 1 jobs passed in 29 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8618a3437b6fca76e6f909e8aad4fe0b17da0649">8618a3437b6fca76e6f909e8aad4fe0b17da0649</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8618a3437b6fca76e6f909e8aad4fe0b17da0649">8618a3437b6fca76e6f909e8aad4fe0b17da0649</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2136839365">EE Development Acceptance (master) - 4139 - 2022-12-21[12:38:13]</a></p><details><summary><strong>ci:test:bundle - 1 out of 1 jobs PASSED</strong></summary></details><h5>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6516/jenkins-report.html">here</a>.</h5></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-26/jobs/test-portal-acceptance-pullrequest(master)/builds/6516/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-26/jobs/test-portal-acceptance-pullrequest(master)/builds/6516/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-26/jobs/test-portal-acceptance-pullrequest(master)/builds/6516/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/6516/">test-portal-acceptance-pullrequest(master)#6516</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6516/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>bundle<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/141">JoyceWang08211#141</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2136865891">[master] ci:test:bundle - JoyceWang08211 &gt; JoyceWang08211 - PR#141 - 2022-12-21[21:55:17]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-26.liferay.com/job/publish-testray-report/15860/">publish-testray-report#15860</a><br/></div></div>

JoyceWang08211 commented:
ci:test:saml

liferay-continuous-integration commented:
<html><h3>:x: ci:test:saml - 0 out of 2 jobs passed in 42 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4140">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:132null_7567</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/72437f3b626b91d8ebbeb2380e27148a0d5dc40d">72437f3b626b91d8ebbeb2380e27148a0d5dc40d</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8618a3437b6fca76e6f909e8aad4fe0b17da0649">8618a3437b6fca76e6f909e8aad4fe0b17da0649</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2136839365">EE Development Acceptance (master) - 4139 - 2022-12-21[12:38:13]</a></p><details><summary><strong>ci:test:saml - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/7567/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/374016/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7567/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/374016//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/374016//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort_">LocalFile.SAML#ImportJSONFileToConfigureSAMLSPOnNonDefaultPort</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-32/test-portal-acceptance-pullrequest(master)/7567/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-32/test-portal-acceptance-pullrequest(master)/7567/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-32/test-portal-acceptance-pullrequest(master)/7567/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest(master)/7567/">test-portal-acceptance-pullrequest(master)#7567</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7567/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>saml<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/141">JoyceWang08211#141</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2136891013">[master] ci:test:saml - JoyceWang08211 &gt; JoyceWang08211 - PR#141 - 2022-12-21[23:00:12]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-32.liferay.com/job/publish-testray-report/18877/">publish-testray-report#18877</a><br/></div></div>

JoyceWang08211 commented:
ci:test:saml

liferay-continuous-integration commented:
<html><h3>:x: ci:test:saml - 0 out of 2 jobs passed in 48 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4140">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:138null_5388</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/72437f3b626b91d8ebbeb2380e27148a0d5dc40d">72437f3b626b91d8ebbeb2380e27148a0d5dc40d</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8618a3437b6fca76e6f909e8aad4fe0b17da0649">8618a3437b6fca76e6f909e8aad4fe0b17da0649</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2136839365">EE Development Acceptance (master) - 4139 - 2022-12-21[12:38:13]</a></p><details><summary><strong>ci:test:saml - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest(master)/5388/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/365741/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5388/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/365741//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/365741//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort_">LocalFile.SAML#ImportJSONFileToConfigureSAMLSPOnNonDefaultPort</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-38/test-portal-acceptance-pullrequest(master)/5388/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-38/test-portal-acceptance-pullrequest(master)/5388/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-38/test-portal-acceptance-pullrequest(master)/5388/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest(master)/5388/">test-portal-acceptance-pullrequest(master)#5388</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5388/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>saml<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/141">JoyceWang08211#141</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2136901480">[master] ci:test:saml - JoyceWang08211 &gt; JoyceWang08211 - PR#141 - 2022-12-22[00:24:35]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-38.liferay.com/job/publish-testray-report/12008/">publish-testray-report#12008</a><br/></div></div>

JoyceWang08211 committed:
LPS-170861 Add target to deploy keystore on non-default port

JoyceWang08211 committed:
LPS-170861 Add test to import JSON file to configure SAML SP on non-default port

JoyceWang08211 committed:
LPS-170861 CI Test

JoyceWang08211 commented:
ci:test:saml

liferay-continuous-integration commented:
<html><h3>:x: ci:test:saml - 0 out of 2 jobs passed in 42 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4145">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:137null_7383</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/6d36327a0a9f649a244b829bc9083c4fcf0255ad">6d36327a0a9f649a244b829bc9083c4fcf0255ad</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7908b6017276afa5aefa618d13d53ef5c3934cf1">7908b6017276afa5aefa618d13d53ef5c3934cf1</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2137987556">EE Development Acceptance (master) - 4144 - 2022-12-22[15:09:48]</a></p><details><summary><strong>ci:test:saml - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7383/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/395704/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7383/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/395704//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/395704//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort_">LocalFile.SAML#ImportJSONFileToConfigureSAMLSPOnNonDefaultPort</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7383/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7383/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7383/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7383/">test-portal-acceptance-pullrequest(master)#7383</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7383/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>saml<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/141">JoyceWang08211#141</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2142132676">[master] ci:test:saml - JoyceWang08211 &gt; JoyceWang08211 - PR#141 - 2022-12-22[19:11:32]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/16980/">publish-testray-report#16980</a><br/></div></div>

JoyceWang08211 committed:
LPS-170861 TF

JoyceWang08211 commented:
ci:test:saml

liferay-continuous-integration commented:
<html><h3>:x: ci:test:saml - 0 out of 2 jobs passed in 57 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1e73c2ea84a3ae0e68eef580ec1019577f1c18ba">1e73c2ea84a3ae0e68eef580ec1019577f1c18ba</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1e73c2ea84a3ae0e68eef580ec1019577f1c18ba">1e73c2ea84a3ae0e68eef580ec1019577f1c18ba</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2142289279">EE Development Acceptance (master) - 4146 - 2022-12-22[20:17:07]</a></p><details><summary><strong>ci:test:saml - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7385/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/323613/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7385/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/323613//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/323613//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort_">LocalFile.SAML#ImportJSONFileToConfigureSAMLSPOnNonDefaultPort</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7385/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7385/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.SAML_ImportJSONFileToConfigureSAMLSPOnNonDefaultPort/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2022-12/test-1-37/test-portal-acceptance-pullrequest(master)/7385/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7385/">test-portal-acceptance-pullrequest(master)#7385</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7385/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>saml<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/JoyceWang08211/liferay-portal/pull/141">JoyceWang08211#141</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2142454217">[master] ci:test:saml - JoyceWang08211 &gt; JoyceWang08211 - PR#141 - 2022-12-23[00:04:45]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/16987/">publish-testray-report#16987</a><br/></div></div>
